packages/web-app-files/src/views/shares/SharedResource.vue

```diff
@@ -169,6 +169,10 @@ export default defineComponent({
 
     breadcrumbs() {
       return concatBreadcrumbs(
+        {
+          text: this.$gettext('Shares'),
```

Comment:
Wait no, this can only happen if sharing jail is there (oCIS feature)!

Reply:
Ok, will add a check later, just sitting on other stuff right now

Reply:
Check not needed. The view only gets navigated to in ocis with sharin jail. Oc10 navigates into mounted share in "All files" view.